### PR TITLE
Fix a handful of link targets in the BigAnimal docs

### PR DESCRIPTION
**Pull request by josh-heyer**
## What Changed?

Update a few links where typos or section name changes have invalidated the target

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx [hunk 1]
```diff
@@ -100,7 +100,7 @@ Provision your cluster before configuring IAM for Postgres.
     1. On the **Additional Settings** tab, under **Authentication**, select **Identity and Access Management (IAM) Authentication**. 
     1. Select **Create Cluster** or **Save**.
     !!!note
-    To turn on IAM authentication using the CLI, see the [Using IAM authentication on AWS](/biganimal/latest/reference/cli/#using-iam-authentication-on-aws) section in the Using the BigAnimal CLI topic.
+    To turn on IAM authentication using the CLI, see the [Using IAM authentication on AWS](/biganimal/latest/reference/cli/using_features/#iam-authentication-cli-commands) section in the Using the BigAnimal CLI topic.
```
Comment (djw-m) [inline product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx:103]: Loving the loop - that links to a short section that ends with "Find out more about IAM for Postgres" and links back here.

**Review by djw-m — DISMISSED** [review]
Alll checked locally for correctness - LGTM

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx (hunk 1 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx:103]: I'm open to suggestions. That link goes to the section we have in our measly CLI topics and is specific about the CLI commands. I saw no harm in linking to our measly IAM section in the _01postgres_access topic, which is all we have in terms of our IAM support at the moment. The IAM support itself currently is fairly measly too. My thinking was if a user started with the CLI topic, we should provide a link for more info, but maybe that is not useful. BTW, I use that same model for many of the CLI sections, where I link back to topics that have more information about the feature.

Comment (djw-m) [reply product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx:103]: I don't think it's a problem for us. This is just an edge case where the link back is so close to the landing site.

**Comment by josh-heyer** [comment]
> @josh-heyer how did you find these broken links? I don't think they were caused by recent changes and I haven't seen them show up in the PDF builds.

You'll see them now: https://github.com/EnterpriseDB/docs/actions/runs/4755978473/jobs/8450882012#step:12:35

I fixed a bug in the PDF link-checker that caused it to miss these.

**Pull request merged**